c/windows.c

```diff
@@ -242,33 +240,46 @@ static IUnknown *s_CreateInstance(CLSID *pCLSID, IID *iid) {
 }
 
 static ptr s_ErrorString(DWORD dwMessageId) {
-    char *lpMsgBuf;
+    char *str = s_ErrorStringImp(dwMessageId, NULL);
+    ptr result = Sstring(str);
+    free(str);
+    return result;
+}
+
+static char *s_ErrorStringImp(DWORD dwMessageId, const char *lpcDefault) {
+    wchar_t *lpMsgBuf;
     DWORD len;
-    ptr result;
+    char *result;
+    char *endstr;
 
-    len = FormatMessage(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
-                        NULL, dwMessageId, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), (LPTSTR)&lpMsgBuf, 0, NULL);
-    /* If FormatMessage fails, use the error code in hexadecimal. */
+    len = FormatMessageW(FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
+                        NULL, dwMessageId, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT), (LPWSTR)&lpMsgBuf, 0, NULL);
```

Comment:
Please add another space to indent this line.

Reply:
Fixed.